keydata_test.go

```diff
@@ -1067,27 +1083,325 @@ func (s *keyDataSuite) TestChangePassphraseWrongPassphrase(c *C) {
 	s.checkKeyDataJSONAuthModePassphrase(c, keyData, protected, 0, "12345678", kdfOptions)
 }
 
-type testSnapModelAuthData struct {
-	alg        crypto.Hash
-	authModels []SnapModel
+func (s *keyDataSuite) TestSnapModelAuthErrorHandling(c *C) {
```

Comment:
should this test name also contain Legacy?

Reply:
I renamed it and moved it further down under the legacy tests section.